nodes/scheduling/nodes-scheduler-node-affinity.adoc

```diff
@@ -30,4 +30,4 @@ include::modules/nodes-scheduler-node-affinity-example.adoc[leveloffset=+1]
 == Additional resources
 
 For information about changing node labels, see
-xref:../../nodes/nodes/nodes-nodes-working.adoc#nodes-nodes-working-updating_nodes-nodes-working[Understanding how to update labels on nodes].
+xref:../nodes/nodes-nodes-working.adoc#nodes-nodes-working-updating_nodes-nodes-working[Understanding how to update labels on nodes].
```

Comment:
I can't see that this file is actually used anywhere. But also I think this would probably 404 like the other files with this change.